### PR TITLE
chore: add lint cache

### PR DESCRIPTION
**Pull request by Simon-He95**
[[中文版模板 / Chinese template](https://github.com/alibaba/hooks/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: packages/hooks/src/useKeyPress/index.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export type Options = {
 };
 
 // 键盘事件 keyCode 别名
-const aliasKeyCodeMap = {
+const aliasKeyCodeMap: any = {
```
Comment (crazylxr) [inline packages/hooks/src/useKeyPress/index.ts]: 这个 any 是为了啥？

File: packages/hooks/src/useKeyPress/index.ts [hunk 2]
```diff
@@ -124,9 +124,9 @@ const aliasKeyCodeMap = {
 };
 
 if (isAppleDevice) {
-  aliasKeyCodeMap['meta'] = [91, 93];
+  aliasKeyCodeMap.meta = [91, 93];
```
Comment (crazylxr) [inline packages/hooks/src/useKeyPress/index.ts]: 这个改动我认为几乎没有什么意义呢

Comment (Simon-He95) [reply packages/hooks/src/useKeyPress/index.ts]: 这个是lint格式化, 原本属性没有meta才加了一个any

Comment (crazylxr) [reply packages/hooks/src/useKeyPress/index.ts]: 我觉得可以不用改这块代码，要改的话就好好加一个类型吧，别用 any。

Comment (Simon-He95) [reply packages/hooks/src/useKeyPress/index.ts]: ok, done

File: packages/hooks/src/useKeyPress/index.ts [hunk 3]
```diff
@@ -136,7 +137,7 @@ const modifierKey = {
   alt: (event: KeyboardEvent) => event.altKey,
   meta: (event: KeyboardEvent) => {
     if (event.type === 'keyup') {
-      return aliasKeyCodeMap['meta'].includes(event.keyCode);
+      return aliasKeyCodeMap.meta.includes(event.keyCode);
```
Comment (crazylxr) [inline packages/hooks/src/useKeyPress/index.ts]: 这个没有改回来

Comment (Simon-He95) [reply packages/hooks/src/useKeyPress/index.ts]: done

**Review by liuyib — APPROVED** [review]
复原了一些对代码 lint 的处理，看了下代码中还有很多没有改正的 lint warning 代码，所以准备单拉一个 PR 处理 lint warning 的代码

**Comment by liuyib** [comment]
@crazylxr  可以合并了。lint 警告的代码我单拉一个 PR 来处理

**Pull request merged**